### ISSUE TITLE
请问现在的系统实现了 缓存节点的功能了吗

### ISSUE THREAD

**Issue by XinYu-777** [alibaba/graph-learn#172]
请问现在的系统实现缓存节点的功能了吗，找没找到

**Comment by baoleai** [comment]
No, we removed this part of the code in the open source version because the previous coupling was a bit high.
We are designing new feature storage and caching solutions :)

**Comment by XinYu-777** [comment]
> No, we removed this part of the code in the open source version because the previous coupling was a bit high. We are designing new feature storage and caching solutions :)

so, it means that in this version, in every iteration, the worker has to pull all remove needed features local in order to train?
And, when the next version will come out :)
